### PR TITLE
Implement it

### PR DESCRIPTION
**Pull request by trieloff**

### CODE REVIEW

**Comment by trieloff** [comment]
Try https://adobeioruntime.net/api/v1/web/helix/helix-services/purge@ci?host=theblog--adobe.hlx.page&xfh=blog.adobe.com&path=/en/2020/08/14/6-ways-ta-adapt-advance-your-business-during-pandemic.html

File: README.md [hunk 1]
```diff
@@ -15,9 +15,13 @@
 ## Usage
 
 ```bash
-curl https://adobeioruntime.net/api/v1/web/helix/helix-services/purge@v1
+curl -X POST https://adobeioruntime.net/api/v1/web/helix/helix-services/purge@v1?host=…&xfh=…&path=…
 ```
 
+- `host`: the inner cdn hostname
+- `xfh`: the outer cdn hostnames, comma separated
```
Comment (dominique-pfister) [inline README.md:22]: `xfh` === `X-Forwarded-Host` ?

**Review by rofe — CHANGES_REQUESTED** [review]
Nice! But...

> Try https://adobeioruntime.net/api/v1/web/helix/helix-services/purge@ci?host=theblog--adobe.hlx.page&xfh=blog.adobe.com&path=/en/2020/08/14/6-ways-ta-adapt-advance-your-business-during-pandemic.html

... obviously in the case of theblog it's not so simple, because the paths differ between inner and outer CDN... it would be cool if instead of the `path` parameter, one could alternatively specify a surrogate key. This way we could also use this service to bust all cached index json segments at once.

We could then create a "Purge cache" bookmarklet, similar to the "Preview" one, which calculates the surrogate key from the URL of the source document.

**Review by rofe — APPROVED** [review]

**Pull request merged**